src/CSharpExtensions.Analyzers/AddMissingMembersOfTwinTypeCodeFixProvider.cs

```diff
@@ -50,16 +50,34 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         private async Task<Document> AddMissingMembers(Document contextDocument, INamedTypeSymbol namedType, BaseTypeDeclarationSyntax typeDeclaration, TwinTypeInfo twinTypeInfo, CancellationToken token)
         {
             var syntaxGenerator = SyntaxGenerator.GetGenerator(contextDocument);
-            var newMembers =  CreateMissingMembers(namedType, twinTypeInfo, syntaxGenerator).ToArray();
             var newType = typeDeclaration switch
             {
-                TypeDeclarationSyntax td => td.AddMembers(newMembers),
-                EnumDeclarationSyntax ed => ed.AddMembers(newMembers.OfType<EnumMemberDeclarationSyntax>().ToArray()),
+                TypeDeclarationSyntax td => AddMembers(td, namedType, twinTypeInfo, syntaxGenerator),
+                EnumDeclarationSyntax ed => AddEnumMembers(ed, namedType, twinTypeInfo, syntaxGenerator),
                 _ => typeDeclaration
             };
             return await ReplaceNodes(contextDocument, typeDeclaration, newType, token);
         }
 
+        private static TypeDeclarationSyntax AddMembers(TypeDeclarationSyntax td, INamedTypeSymbol namedType, TwinTypeInfo twinTypeInfo, SyntaxGenerator syntaxGenerator)
+        {
+            var newMembers = CreateMissingMembers(namedType, twinTypeInfo, syntaxGenerator).ToArray();
+            return td.AddMembers(newMembers);
+        }
+
+        private static EnumDeclarationSyntax AddEnumMembers(EnumDeclarationSyntax ed, INamedTypeSymbol namedType, TwinTypeInfo twinTypeInfo, SyntaxGenerator syntaxGenerator)
+        {
+            var members = new List<EnumMemberDeclarationSyntax>();
+            var twinMembers = twinTypeInfo.GetTwinMembersFor(namedType);
+            foreach (var twinMember in twinMembers)
+            {
+                SyntaxNode valueNode = twinMember.IsEnumWithValue ? syntaxGenerator.LiteralExpression(twinMember.EnumConstantValue) : null;
+                var enumMember = (EnumMemberDeclarationSyntax)syntaxGenerator.EnumMember(twinMember.Symbol.Name, valueNode).WithAdditionalAnnotations(Formatter.Annotation);
+                members.Add(enumMember);
+            }
+            var newMembers = SyntaxFactory.SeparatedList(members);
+            return ed.WithMembers(newMembers);
```

Comment:
This will completely override existing members instead of adding only missing ones. What if one enum is a subset of its twins?

Reply:
Correct. Will be fixed under the feature flag.